### PR TITLE
docs: update kb0084.md

### PR DESCRIPTION
**Pull request by Nnyny**

### CODE REVIEW

File: knowledge-base/kb0084.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 # Keyman for iPhone and iPad, Keyman for Android: FAQ
 
-### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant
 
 ## Question #1:
 How can I add my own keyboard/font to the Keyman application?
 
 ## Answer:
-Please read the blog [Keyman for Android update 1.2 - Add custom keyboards to your device!](http://blog.tavultesoft.com/2014/04/keyman-for-android-update-12-add-custom-keyboards-to-your-device.html) You can also send us the keyboard and we can add it to our repository.
+Please read the articles: [Installing Custom Keyboards to your Android Device](https://help.keyman.com/developer/current-version/guides/distribute/install-kmp-android)
```
Comment (darcywong00) [inline knowledge-base/kb0084.md:8]: Wonder if we make these paths relative for staging and locally hosted sites

```suggestion
Please read the articles: [Installing Custom Keyboards to your Android Device](/developer/current-version/guides/distribute/install-kmp-android)
```

File: knowledge-base/kb0084.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
-### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant
 
 ## Question #1:
 How can I add my own keyboard/font to the Keyman application?
 
 ## Answer:
-Please read the blog [Keyman for Android update 1.2 - Add custom keyboards to your device!](http://blog.tavultesoft.com/2014/04/keyman-for-android-update-12-add-custom-keyboards-to-your-device.html) You can also send us the keyboard and we can add it to our repository.
+Please read the articles: [Installing Custom Keyboards to your Android Device](https://help.keyman.com/developer/current-version/guides/distribute/install-kmp-android) 
+and [Installing Custom Keyboards to your iPhone or iPad](https://help.keyman.com/developer/current-version/guides/distribute/install-kmp-ios).
```
Comment (darcywong00) [inline knowledge-base/kb0084.md:9]:
```suggestion
and [Installing Custom Keyboards to your iPhone or iPad](/developer/current-version/guides/distribute/install-kmp-ios).
```

File: knowledge-base/kb0084.md (hunk 1 above)
Comment (Nnyny) [reply knowledge-base/kb0084.md:8]: Sorry I forgot about this!

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**